tests/unit/http_/test_router.py

```diff
@@ -176,6 +183,100 @@ def test_port_host_dispatcher(self):
                 )
             )
 
+    def test_path_converter(self):
```

Comment:
The `path` converter comes with Werkzeug, which means these tests are more or less to define / explore the exact behavior of the builtin Werkzeug converter, right?

Reply:
essentially, yes.this was also the reason why we never added tests like this before, since it's just testing werkzeug. but they clearly document the behavior of the integration now.